process-engine/bpmn-studio
# 🐛 Fix ErrorEndEvent colorization

## Changes

1. Fix colorization of `ErrorEndEvents`
2. Add `onProcessError` event listeners to LET
3. Fix some vulnerabilities through `npm audit`
4. Add `config` files to app resources (required for the runtime Post-Migrations)

## Issues

Closes #1800 
Part of https://github.com/process-engine/process_engine_runtime/issues/400

PR: #1801

## How to test the changes

- Run a Process that uses an `ErrorEndEvent`
- See that everything is colorized correctly
